clients/cobol-lsp-vscode-extension/snippets.json

```diff
@@ -2331,15 +2331,15 @@
         "body": [
             "ROW START ${1:TBL|TBFnam}."
         ],
-        "description": "[DaCo] ROW START",
+        "description": "[DaCo] Set the line-currency (RCU) to zero in order to be able to process the  \n table sequentially from the beginning.\n After this command is given, the ROW NEXT command can be used to move the\n currency 1 line forward.\n",
         "scope": "cobol"
     },
     "TABLE ROW SAVE": {
         "prefix": "ROW SAVE",
         "body": [
             "ROW SAVE ${1:TBLnam} IN ${2:value}."
         ],
-        "description": "[DaCo] ROW SAVE",
+        "description": "[DaCo] Place the TB-1 in TB-n with the offered value and save the RCU of this line.\n The offered value must be 2, 3 or 4.\n The TB-n table lines can be used in the program logic (eg calculate manupulations) involving multiple table lines.\n",
```

Comment:
I think I am just missing some context here
[ "ROW SAVE ${1:TBLnam} IN ${2:value}." ],
Place the TB-1 in TB-n with the offered value and save the RCU of this line. The offered value must be 2, 3 or 4. The TB-n table lines can be used in the program logic (eg calculate manupulations) involving multiple table lines.

- What are TB-1 and TB-n? Do 1 and n correspond to the two variables in the code?
- I guess that value that has to be 2,3,4 is what is meant by the "offered value"?
- What is being saved here, a table or a table row? Are there only 3 slots available to save them hence the limitation on the value
- Table lines = rows?
- Is it "program logic involving multiple table lines" or "calculate manipulations involving multiple table lines"?

Reply:
@zeibura Same here as well. I will confirm and update you.

Reply:
Place the content of the Table Buffer-1 in Table Buffer-n, where n is 2, 3 or 4. The three Table Buffers (2, 3 and 4) can be used in the program logic.

Reply:
@zeibura Is below text ok? We would like to explain what <value>  is here.

Place the content of Table Buffer-1 in Table Buffer-<value>, where <value> is 2, 3 or 4. The three Table Buffers (2, 3 and 4) can be used in the program logic.

Reply:
Done